src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -280,6 +289,12 @@ private SmugglerProgressBase.Counts ProcessIndexes(SmugglerResult result)
                                 continue;
                             }
 
+                            if (indexDefinition.Name.StartsWith("Auto/", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why is that? What about an export from a 4.0 system that has auto indexes?

Reply:
I think that's ok. This only for static indexes (in particular old auto indexes having Linq def):

https://github.com/grisha-kotler/ravendb/blob/c864b4a29d06334478638e5b4bb47fa84069c476/src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs#L277-L292

Auto 4.0 indexes are handled here:
https://github.com/grisha-kotler/ravendb/blob/c864b4a29d06334478638e5b4bb47fa84069c476/src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs#L252-L265

Reply:
Just exported / imported db with auto index. It's fine.